src/imitation/algorithms/mce_irl.py

```diff
@@ -383,6 +394,7 @@ def set_demonstrations(self, demonstrations: MCEDemonstrations) -> None:
         if isinstance(demonstrations, Iterable):
             first_item, demonstrations = util.get_first_iter_element(demonstrations)
             if isinstance(first_item, types.Trajectory):
+                demonstrations = cast(Iterable[types.Trajectory], demonstrations)
```

Comment:
Copying from https://github.com/HumanCompatibleAI/imitation/pull/594/files (by @dan-pandori):
```suggestion
                # Safe since we know the first item is a Trajectory and `demonstrations`
                # is an Iterable
                demonstrations = cast(Iterable[types.Trajectory], demonstrations)
```

Reply:
I've found a better way to do this, I think